client/src/components/product-questions/SearchQuestions.jsx

```diff
@@ -0,0 +1,39 @@
+import React, { useState, useEffect } from 'react';
+
+class SearchQuestions extends React.Component {
+
+  constructor(props) {
+    super(props)
+
+    this.state = {
+        searchText: '',
+    };
+
+    this.handleChange = this.handleChange.bind(this);
+  }
+
+  handleChange(event) {
+    const newSearchText = event.target.value;
+    this.setState({
+      searchText: newSearchText
+    });
+
+    this.props.handlSearchTextChanged(newSearchText);
```

Comment:
consider a debounce function to group bursts of events into 1

Reply:
thank you! will look into this